<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieSearchWidget</class>
 <widget class="QWidget" name="MovieSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scraper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboScraper"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rBtnSearchTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Activate to search by movie title.</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyLineEdit" name="searchStringTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the movie title you wish to search for.
Press the Enter key to start the search.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnSearchId">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Activate to search by remote database id.</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyLineEdit" name="searchStringId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>When using IMDB you can also use the IMDB id as search query.
If you want to search by an TMDB or OFDb id please prefix it with &quot;id&quot;.
Press the Enter key to start the search.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QTableWidget" name="results">
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(243, 246, 250);
color: #395265;
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #4185b6, stop:1 #1b6aa5);
</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Result</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 1px solid rgba(0, 0, 0, 100);
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 200);
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Infos to load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkTitle">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkTagline">
          <property name="text">
           <string>Tagline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkRating">
          <property name="text">
           <string>Rating</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkReleased">
          <property name="text">
           <string>Released</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkRuntime">
          <property name="text">
           <string>Runtime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkDirector">
          <property name="text">
           <string>Director</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkWriter">
          <property name="text">
           <string>Writer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkCertification">
          <property name="text">
           <string>Certification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkTrailer">
          <property name="text">
           <string>Trailer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkOverview">
          <property name="text">
           <string>Overview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkPoster">
          <property name="text">
           <string>Poster</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkBackdrop">
          <property name="text">
           <string>Backdrop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkActors">
          <property name="text">
           <string>Actors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkGenres">
          <property name="text">
           <string>Genres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkStudios">
          <property name="text">
           <string>Studios</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkCountries">
          <property name="text">
           <string>Countries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkTags">
          <property name="text">
           <string>Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkLogo">
          <property name="text">
           <string>Logo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkClearArt">
          <property name="text">
           <string>Clear Art</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkCdArt">
          <property name="text">
           <string>CD Art</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkBanner">
          <property name="text">
           <string>Banner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkThumb">
          <property name="text">
           <string>Thumb</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkUnCheckAll">
          <property name="text">
           <string>(Un)Check all</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">smallWidgets/MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>smallWidgets/MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
